pkg/cmd/roachtest/tests/network.go

```diff
@@ -424,3 +436,22 @@ func registerNetwork(r registry.Registry) {
 		Run:              runClientNetworkConnectionTimeout,
 	})
 }
+
+// iptablesPacketsDropped returns the number of packets dropped to a given node due to an iptables rule.
+func iptablesPacketsDropped(
+	ctx context.Context, l *logger.Logger, c cluster.Cluster, node option.NodeListOption,
+) (int, error) {
+	res, err := c.RunWithDetailsSingleNode(ctx, l, option.WithNodes(node), "sudo iptables -L -v -n")
+	if err != nil {
+		return 0, err
+	}
+	rows := strings.Split(res.Stdout, "\n")
+	// iptables -L outputs rows in the order of: chain, fields, and then values.
+	// We care about the values so only look at row 2.
+	values := strings.Fields(rows[2])
+	if len(values) == 0 {
+		return 0, errors.Errorf("no configured iptables rules found")
+	}
+	packetsDropped, err := strconv.Atoi(values[0])
+	return packetsDropped, err
```

Comment:
For debuggability, it would be helpful to include `iptables`'s output in the error message.